docs/experimental-features.md

```diff
@@ -439,20 +439,22 @@ When it is enabled:
 - IPNS publishers push records to a name-specific pubsub topic,
   in addition to publishing to the DHT.
 - IPNS resolvers subscribe to the name-specific topic on first
-  resolution and receive subsequently published records through pubsub in real time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.
+  resolution and receive subsequently published records through pubsub in real time.
+  This makes subsequent resolutions instant, as they are resolved through the local cache.
 
 Both the publisher and the resolver nodes need to have the feature enabled for it to work effectively.
 
+Note: While IPNS pubsub has been available since 0.4.14, it received major changes in 0.5.0.
+Users interested in this feature should upgrade to at least 0.5.0
+
 ### How to enable
 
 run your daemon with the `--enable-namesys-pubsub` flag; enables pubsub.
 
 ### Road to being a real feature
 
 - [ ] Needs more people to use and report on how well it works
-- [ ] Add a mechanism for last record distribution on subscription,
-  so that we don't have to hit the DHT for the initial resolution.
-  Alternatively, we could republish the last record periodically.
+- [ ] Pubsub enabled as a real feature
```

Comment:
@Stebalien Is there also a requirement here that we have a plan for dealing with people using different versions of the protocol when there are future upgrades, or is that just a standard issue we have to deal with even for real features?

Reply:
Eh, that's a standard issue.